oscillatord/monitoring.go

```diff
@@ -216,6 +216,7 @@ type GNSS struct {
 	LSChange        LeapSecondChange `json:"lsChange"`
 	LeapSeconds     int              `json:"leap_seconds"`
 	SatellitesCount int              `json:"satellites_count"`
+	SurveyInError   int              `json:"survey_in_position_error"`
```

Comment:
Let's maybe use full name here? Like `SurveyInPositionErrror`.
Because `SurveyIn` doesn't make much sense to me.

Reply:
Either like it's in `oscillatord` or like in `gpsd` (3D error).